.github/workflows/release_mac.yml

```diff
@@ -0,0 +1,50 @@
+name: Mac
+
+on: [push]
```

Comment:
just `push` wont work AFAICS

Reply:
Yes. trying to identify the correct event to build on a new commit to this pr